spa/src/elements/buttons/CircleButton.js

```diff
@@ -11,6 +15,18 @@ function CircleButton({ icon, type, color, onClick, disabled, loading, children,
       )}
     </S.CircleButton>
   );
+
+  if (tootTipText) {
+    return (
+      <S.CircleButtonDiv>
```

Comment:
Better to just add a div instead of styled component
```suggestion
      <div>
```


Did you try removing this div altogether?
```suggestion
```

Reply:
updated.